python/tskit/trees.py

```diff
@@ -2242,6 +2242,22 @@ def num_migrations(self):
         """
         return self._ll_tree_sequence.get_num_migrations()
 
+    @property
+    def oldest_time(self):
+        """
+        Returns the oldest time for any of the edges in this tree sequence.
+
+        :return: The oldest time in this tree sequence.
+        :rtype: float
+        """
+        # Edges are guaranteed to be listed in parent-time order, so we can just get the
+        # last one, assuming there are any edges (otherwise return None). This is O(1)
+        # compared to looking for the maximum node time.
+        if self.num_edges:
```

Comment:
Using Falseyness can have nasty gotchas so I strongly discourage it. Much more readable to explicitly state what you're guarding against here.

Reply:
Good point